app/components/bestuurseenheid-select.js

```diff
@@ -36,11 +36,11 @@ export default Component.extend({
       filter: { naam: term }
     });
   }),
-
+  
```

Comment:
We should remove those spaces no ?

Reply:
Not sure.

Reply:
Doesn't have a meaning, so should be removed indeed.

Reply:
I'm the pedantic one.  It's a tiny thing, but I prefer there to be no spaces on empty lines.  Consistency is awesome and no spaces makes a tiny bit more sense as a default.  Hence: no spaces on empty lines.